<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 7.0.1 (Python 3.12.2 on win32)" generated="2024-06-28T13:37:49.437857" rpa="false" schemaversion="5">
<suite id="s1" name="Calculator" source="c:\Iterasys\Iterasys2\lISTA14C\pages\calculator.robot">
<test id="s1-t1" name="Test Subtraction" line="13">
<kw name="Setup Appium">
<msg time="2024-06-28T13:37:49.488563" level="FAIL">No keyword with name 'Setup Appium' found.</msg>
<status status="FAIL" start="2024-06-28T13:37:49.487566" elapsed="0.002992">No keyword with name 'Setup Appium' found.</status>
</kw>
<kw name="Press Key">
<arg>5</arg>
<status status="NOT RUN" start="2024-06-28T13:37:49.491556" elapsed="0.000000"/>
</kw>
<kw name="Press Key">
<arg>-</arg>
<status status="NOT RUN" start="2024-06-28T13:37:49.492554" elapsed="0.000997"/>
</kw>
<kw name="Press Key">
<arg>3</arg>
<status status="NOT RUN" start="2024-06-28T13:37:49.496544" elapsed="0.000000"/>
</kw>
<kw name="Press Key">
<arg>=</arg>
<status status="NOT RUN" start="2024-06-28T13:37:49.522473" elapsed="0.000997"/>
</kw>
<doc>Testa a função de subtração na calculadora</doc>
<status status="FAIL" start="2024-06-28T13:37:49.485571" elapsed="0.044007">No keyword with name 'Setup Appium' found.</status>
</test>
<status status="FAIL" start="2024-06-28T13:37:49.445205" elapsed="0.087811"/>
</suite>
<statistics>
<total>
<stat pass="0" fail="1" skip="0">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat pass="0" fail="1" skip="0" id="s1" name="Calculator">Calculator</stat>
</suite>
</statistics>
<errors>
<msg time="2024-06-28T13:37:49.482612" level="ERROR">Error in file 'c:\Iterasys\Iterasys2\lISTA14C\pages\calculator.robot' on line 2: Resource file '..\configs\variables.robot' does not exist.</msg>
</errors>
</robot>
